src/components/notification/notification.ts

```diff
@@ -0,0 +1,66 @@
+import { unsafeCSS, html, LitElement } from 'lit';
+import { customElement, property, state } from 'lit/decorators.js';
+import { Control } from 'ol/control';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+
+import style from '../../styles/notification.css?inline';
+import theme from '../../styles/theme.css?inline';
+
+import SVGCreator from '../../utils/svg-creator';
+import NotificationElement from '../../types/notification-element';
+
+@customElement('notification-box')
+class NotificationBox extends LitElement {
+  @property()
+  notification: NotificationElement = { type: '', message: '', rule: {} };
+  @property()
+  mode: string = 'light';
+
+  @state() icon = '';
+  @state() theme = '';
+
+  static styles = [unsafeCSS(theme), unsafeCSS(style)];
+
+  constructor() {
+    super();
+  }
+
+  firstUpdated(): void {
+    switch (this.notification.type) {
+      case 'info':
+        this.icon = SVGCreator.info;
+        this.theme = this.mode === 'light' ? 'notification-element-info-light' : 'notification-element-info-dark';
```

Comment:
```suggestion
        this.theme = `notification-element-info-${this.mode}`;
```
or probably better to add a class per type and mode, the css will be clearer, default could be light mode
```
.warning {
 &.dark{
 ...
 }
}
```
also since css is scoped by lit element no need of such complicated css names

Reply:
The first way has been applied